modules/fedimint-ln-common/src/lib.rs

```diff
@@ -654,3 +654,27 @@ pub struct PrunedInvoice {
     pub timestamp: SystemTime,
     pub expiry: Duration,
 }
+
+impl TryFrom<Bolt11Invoice> for PrunedInvoice {
+    type Error = anyhow::Error;
+
+    fn try_from(invoice: Bolt11Invoice) -> Result<Self, Self::Error> {
+        Ok(PrunedInvoice {
```

Comment:
if you're just wrapping in an `Ok()` can just use do `Impl From<...`

Reply:
Since this becomes a public interface I didn't want to assume the invoice having an amount, which this conversion requires. 

https://github.com/fedimint/fedimint/blob/54006ee0b89244529c1943dbfc4237b418a6c3b3/modules/fedimint-ln-common/src/lib.rs#L664-L666

Reply:
Ah missed that